kafka-client/src/main/java/dev/responsive/kafka/internal/stores/CommitBuffer.java

```diff
@@ -409,6 +410,18 @@ private boolean triggerFlush() {
   }
 
   public void flush(final long consumedOffset) {
+    if (consumedOffset < lastFlushedOffset) {
+      log.error("trying to commit an offset {} older than last flushed {}",
```

Comment:
when can this happen? Can we make this error message more actionable (or is it safe to ignore/just got fenced?)